common/DiffMatchPatch.java

```diff
@@ -31,91 +31,74 @@
  * Applies the patch onto another text, allowing for errors.
  *
  * @author fraser@google.com (Neil Fraser)
- * 
+ *
  * Source: https://github.com/google/diff-match-patch
  * TODO: Add diff-patch-match to WORKSPACE and use directly instead of copying the code.
  */
 
-/**
- * Class containing the diff, match and patch methods.
- * Also contains the behaviour settings.
- */
+/** Class containing the diff, match and patch methods. Also contains the behaviour settings. */
 public class DiffMatchPatch {
```

Comment:
Looks like package name is incorrect. Is it ok?
Maybe the name should be `com.google.startupos.common`?

Reply:
Maybe replace `import java.util.*` to single class import?

Reply:
As noted, this code is taken as-is from another repo.